src/bin/vip-search-replace.js

```diff
@@ -12,33 +12,39 @@ const debug = debugLib( '@automattic/vip:bin:vip-search-replace' );
 const examples = [
 	// `search-replace` flag
 	{
-		usage: 'vip search-replace <file.sql> --search-replace="from,to"',
+		usage: 'vip search-replace file.sql --search-replace="from,to"',
 		description:
-			'Replace instances of <from> with <to> in the provided <file.sql>\n' +
-			'       * Ensure there are no spaces between your search-replace parameters',
+			'Search for every instance of the value "from" in the local input file named "file.sql" and replace it with the value "to".\n' +
+			'       * Results of the operation output to STDOUT by default.',
 	},
 	// `in-place` flag
 	{
-		usage: 'vip search-replace <file.sql> --search-replace="from,to" --in-place',
-		description: 'Perform Search and Replace explicitly on the provided input <file.sql> file',
+		usage: 'vip search-replace file.sql --search-replace="from,to" --in-place',
+		description:
+			'Perform the search and replace operation and save the results to the local input file "file.sql".',
```

Comment:
Should we mention that the S&R operation is performed on `file.sql`?

Reply:
I'm having a hard time understanding what needs to be improved in order to address your concern.

Is this wording a bit better?
`Perform the search and replace operation on the local input file "file.sql" and overwrite the file with the results.`

If not, I welcome a version of the description from you that is clearer. 🙏